srcpkgs/sqlite-replication/template

```diff
@@ -29,15 +28,21 @@ pre_configure() {
 	sed -i -e 's/ -ltinfo//g' configure
 }
 
+post_install() {
+	vmkdir usr/lib/pkgconfig
+	mv "$DESTDIR"/usr/lib{/sqlite-replication,}/pkgconfig/sqlite3.pc
+	# Alpine just deletes the binary, but someone may want it for debugging
+	mv "$DESTDIR"/usr/bin/sqlite3{,.sqlite-replication}
+}
+
 sqlite-replication-devel_package() {
 	depends="sqlite-replication>=${version}_${revision}"
-	replaces="sqlite-devel>=0"
-	provides="sqlite-devel-${version}_${revision}"
+	conflicts="sqlite-devel>=0"
```

Comment:
Why does this conflict? Isn't the point of this PR to make sqlite-replication not conflict with sqlite? Or is -devel exempt?

Reply:
Cannot link to both at once, so I do not care about the -devel packages conflicting.